### PR TITLE
Add test cases for NDA renewal feature

### PR DESCRIPTION
**Pull request by viorelaioia**

### CODE REVIEW

**Comment by viorelaioia** [comment]
@johngian , can you please review this?

File: docs/testing.rst [hunk 1]
```diff
@@ -35,3 +35,160 @@ package run::
...
+
+Test Cases for NDA renewal feature
+----------------------------------
+
+**Verify that the inviter, who is also a group curator is able to renew a user membership 2 weeks before it expires**
+    **Preconditions:**
+        #. Create a closed group with terms in mozillians.org
+        #. Set the membership to expire after several days (>14)
+        #. Login as group curator and invite a mozillian to the closed group
+        #. Login as the mozillian who was invited by curator and accept the invitation to the group and the terms (The mozillians becomes a group member)
```
Comment (johngian) [inline docs/testing.rst]: Please add a sub-bullet for the ``( The mozillians ...)``. Also its `` Mozillian becomes a group member``.

File: docs/testing.rst [hunk 2]
```diff
@@ -35,3 +35,160 @@ package run::
...
+    **Steps:**
+        #. Verify that the group member will receive an email notification 2 weeks before his membership expires
```
Comment (johngian) [inline docs/testing.rst]: ``his`` --> ``their``

File: docs/testing.rst [hunk 3]
```diff
@@ -35,3 +35,160 @@ package run::
...
+        #. Verify that the inviter will receive an email notification 2 weeks before user's membership expires
+        #. Login to mozillians.org as the inviter( and also group curator)
```
Comment (johngian) [inline docs/testing.rst]: ``inviter (and also...``

File: docs/testing.rst [hunk 4]
```diff
@@ -35,3 +35,160 @@ package run::
...
+    **Expected:**
+        The user continues to be a member of the group, as the membership was renewed by inviter
+
+
+**Verify that a group curator is able to renew a user membership 2 weeks before it expires, if the inviter is no longer a curator**
+    **Preconditions:**
+        #. Create a reviewed group with terms
+        #. Set the membership to expire after several days (>14)
+        #. Login as group curator and invite a mozillian to the closed group
+        #. Login as the mozillian who was invited by curator and accept the invitation to the group and the terms (The mozillians becomes a group member)
```
Comment (johngian) [inline docs/testing.rst]: Same here.

File: docs/testing.rst [hunk 5]
```diff
@@ -35,3 +35,160 @@ package run::
...
+        #. Add a new curator to the group and remove the initial curator(who is also the inviter) from the group curators list
```
Comment (johngian) [inline docs/testing.rst]: ``curator (who is ...``)

File: docs/testing.rst [hunk 6]
```diff
@@ -35,3 +35,160 @@ package run::
...
+    **Steps:**
+        #. Verify that the group member will receive an email notification 2 weeks before his membership expires
```
Comment (johngian) [inline docs/testing.rst]: ``his`` --> ``their``

File: docs/testing.rst [hunk 7]
```diff
@@ -35,3 +35,160 @@ package run::
...
+        #. Verify that the user added in precondition is displayed in Group members section
+        #. Click “Confirm Request” for that user
+        #. Select “Pending Terms” option in filter dropdown, then click Filter button
+        #. Verify that the user added in precondition is displayed in Group members section
+    **Expected:**
+        After step 8: The user should receive an email saying “Accepted to Mozillians group “X””
+        After step 10: If the user accepts the terms, he/she will be member of the group again
+
+
+**Verify the status of a user whose request to join a group(with membership set to expire) was never accepted by the curator**
```
Comment (johngian) [inline docs/testing.rst]: ``a group (with ...``

**Pull request merged**